### PR TITLE
Always call Remote.connect() callback

### PR DESCRIPTION
**Pull request by wltsmrz**

### CODE REVIEW

File: src/core/remote.js [hunk 1]
```diff
@@ -421,22 +421,23 @@ Remote.prototype.reconnect = function() {
 /**
  * Connect to the Ripple network
  *
- * @param {Function} [callback]
+ * @param [Function] [callback]
```
Comment (lumberj) [inline src/core/remote.js]: Changed by mistake?

**Comment by lumberj** [comment]
LGTM

**Comment by clark800** [comment]
Might be more clear if the commit message was "Fix: RippleAPI.connect() does not resolve if already connected"

File: src/core/remote.js (hunk 1 above)
Comment (wltsmrz) [inline src/core/remote.js]: Callback is optional

**Comment by wltsmrz** [comment]
I'll add a note in the commit @clark800

**Pull request merged**